pkg/network/config/sysctl/sysctl_darwin.go

```diff
@@ -0,0 +1,69 @@
+// +build darwin
+
+package sysctl
+
+import (
+	"encoding/binary"
+
+	"strings"
+	"syscall"
+	"time"
+)
+
+type sctl struct {
+	ttl      time.Duration
+	lastRead time.Time
+	path     string
+}
+
+func newSCtl(sysctl string, cacheFor time.Duration) *sctl {
+	return &sctl{
+		ttl:  cacheFor,
+		path: sysctl,
+	}
+}
+
+func (s *sctl) get(now time.Time) (string, bool, error) {
```

Comment:
nit: it may be good to have a comment to show what the 3 return vars are (my main concern is that the second one isn't obvious)